plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -699,7 +710,77 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/provisioning_api/api/v[0-9]
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
-  
+
+# Deleting an avatar
+SecRule REQUEST_FILENAME "@endsWith /avatar" \
+    "id:9508616,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+
+# Disabling background
+SecRule REQUEST_FILENAME "@endsWith /apps/theming/background/custom" \
+    "id:9508617,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+
+# When deleting an admin from privacy page
+SecRule REQUEST_FILENAME "@rx /apps/privacy/api/admins/[0-9]*$" \
+    "id:9508618,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+
+# When disabling settings under Settings --> Availability
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/apps/provisioning_api/api/v[0-9]/config/users/dav/user_status_automation$" \
+    "id:9508619,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    setvar:'tx.allowed_methods=%{tx.allowed_methods} DELETE'"
+
+# When configurating an SMTP server under Administration --> Basic Settings --> Email Server
+SecRule REQUEST_FILENAME "@endsWith /settings/admin/mailsettings/credentials" \
+    "id:9508620,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:mail_smtppassword,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0'"
+
+# When removing whitelisted IPs under Administration --> Security --> Brute Force IP whitelsit
+SecRule REQUEST_FILENAME "@rx /apps/bruteforcesettings/ipwhitelist/[0-9]*$" \
```

Comment:
Is there `DELETE` case when the last path segment is not a number?

Reply:
No, the number is needed to distinguish between multiple entries for whitelisted IP addresses.

Reply:
So rather `SecRule REQUEST_FILENAME "@rx /apps/bruteforcesettings/ipwhitelist/[0-9]+$" \` (`+` instead of `*`)?